src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <section class="category-edit">
+    <app-heading :level="1">カテゴリー管理</app-heading>
+    <div class="category-edit__back">
+      <app-router-link
+        block
+        underline
+        key-color
+        hover-opacity
+        to="/categories"
+      >
+        カテゴリー一覧へ戻る
+      </app-router-link>
+    </div>
+    <form class="category-edit__title">
+      <div class="category-edit__title__name">
+        <app-input
+          v-validate="'required'"
+          name="category"
+          type="text"
+          data-vv-as="カテゴリー名"
+          :value="categoryName"
+          @update-value="updateValue"
+        />
+      </div>
+      <div v-if="errorMessage" class="category-edited__notice">
+        <app-text bg-error>{{ errorMessage }}</app-text>
+      </div>
+      <div v-if="doneMessage" class="category-edited__notice">
+        <app-text bg-success>{{ doneMessage }}</app-text>
+      </div>
+    </form>
+    <app-button
+      class="category-edit-submit"
+      button-type="submit"
+      round
+      :disabled="!disabled"
```

Comment:
[imo]

`!disabled`（disabled が false）の時に disabled props が true になるのに違和感を感じます。
`disabled` という変数が true の時に disabled props が true になるというような実装の方が読みやすいので修正をお願いします。

```suggestion
      :disabled="disabled"
```

Reply:
[ミューテーションの不要部分の削除と活性化されてないpropsの定義付け]こちらのコミットでdisabledがtrueの時にdisabled propsがtrueになるように修正いたしました